recipes/mdal/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "mdal" %}
+{% set version = "0.7.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/lutraconsulting/MDAL/archive/0.7.2.tar.gz
+  sha256: 37c050ece0ad2cf0e04f8519f23a4125ab7f88f04f4989de8a2a5daaef846e02
+
+build:
+  number: 0
+  skip: false 
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - cmake
+  host:
+    - libgdal
+  run:
+    - libgdal
+    - hdf5
+    - libnetcdf
+    - libxml2
+
+test:
+  commands:
+    - mdalinfo -h
```

Comment:
Is this also providing a shared library? If so, this should probably use `run_exports` and have a file existence check (see the previously linked recipes).

Reply:
I think I need that clarified?

mdalinfo is not providing a shared library. It is part of the mdal package built above. So it is using the mdal.[dll, so,dylib] shared library.

Reply:
I think I need more clarification on this.

mdalinfo is a part of the package and built by the build scripts. It does not provide a shared library but it does consume the mdal shared library

Reply:
No worries, I tend to be hasty when reviewing for `staged-recipes` so I'm happy to expand when needed :) Do you expect packages could be added to conda-forge which depend on `lib/libmdal.so`? If not, then ignore the rest of this comment.

If yes, then you should add this to the test commands:
```yaml
    - test -f $PREFIX/lib/libmdal$SHLIB_EXT  # [unix]
    - if not exist %LIBRARY_BIN%\\mdal.dll exit 1  # [win]
```
and append to the `build` section the appropriate ABI compatibility pin:
```yaml
build:
  run_exports:
    - {{ pin_subpackage('mdal', max_pin='x.x') }}
```
Conda-build will then add the version constraint to dependent packages like `mdal >=0.7.2,<0.8`. Depending on ABI stability of `mdal` you might need `max_pin='x.x'` or `max_pin='x.x.x'`.

Reply:
Gotcha. I got confused because the comment was directed to the mdalinfo line.

I will add the run_export. The ABI is fairly stable - so it will be x.x.

Thanks for your time and efforts

Reply:
added in next commit